apps/style/applab/style.scss

```diff
@@ -485,10 +485,6 @@ div#visualizationResizeBar {
   text-decoration: underline;
 }
 
-.droplet-dropdown {
-  z-index: 500 !important;
-}
-
```

Comment:
👍 for removing this override, and porting it to the Droplet codebase!

Reply:
The reason we did this before is because we wanted 100% of our changes to go back into droplet `master`. If that is no longer a goal, I agree with this change.

Reply:
I see that we now believe that these changes are compatible with other consumers of droplet, so it does seems possible to merge our changes into `master` at some point.